operator-common/src/main/java/io/strimzi/operator/common/operator/resource/ServiceOperator.java

```diff
@@ -100,7 +101,7 @@ protected void patchNodePorts(Service current, Service desired) {
      * @return
      */
     public Future<Void> hasIngressAddress(String namespace, String name, long pollIntervalMs, long timeoutMs) {
-        return waitFor(namespace, name, pollIntervalMs, timeoutMs, this::isIngressAddressReady);
+        return Util.waitFor(vertx, String.format("%s resource %s in namespace %s", resourceKind, name, namespace), pollIntervalMs, timeoutMs, () -> isIngressAddressReady(namespace, name));
```

Comment:
Similarly here we should pass a `logContext` that says we're waiting for ingress address readiness, and the same below for node port readiness.